util/test/run_tests.py

```diff
@@ -16,7 +16,7 @@
 parser.add_argument('-x', '--test_exclude', default="",
                     help="The tests to exclude, as a regexp filter", type=str)
 parser.add_argument('--in-process',
-                    help="Lists the tests available to run", action="store_true")
+                    help="Do not fork out the tests", action="store_true")
```

Comment:
A better description here would be "Run test code in the same process as test runner", as forking is a linux-only term.

Reply:
Done.